src/components/header/header.jsx

```diff
@@ -58,7 +53,15 @@ const Header = () => {
 							<hr />
 							<li className='wishlist-icon'>My Wishlist</li>
 							<hr />
-							<li onClick={() => logoutHandler()}>Logout 😞</li>
+							<li
+								onClick={() => {
+									logoutHandler();
+									emptyCart();
+									emptyWishlist();
```

Comment:
suggestion: call only one function here and do the rest logic in that function

Reply:
Will do this in the next pr.